exercises/I-floats/exercise.js

```diff
@@ -1,2 +1,9 @@
 var numberOfStudents = 15;
```

Comment:
Incorrect Amount (Optional)

Isn't the correct amount of students and mentors, in your NW4 group - though will let you off. Most importantly, the core implementation is correct